<?xml version="1.0"?>
<launch>

	<arg name="publish_rate" default="100" />
	<arg name="wheel_radius" default="0.0775" />
	<arg name="front_to_rear_wheel_distance" default="0.42" />
	<arg name="left_to_right_wheel_distance" default="0.285" />
	<arg name="min_steering" default="0.003" />
	<arg name="max_steering" default="0.8" />

  	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find jetsoncar_gazebo)/config/rosconsole.conf"/>

	<node name="driver_simulator_node" pkg="jetsoncar_gazebo" type="driver_simulator_node" respawn="true" output="screen">
		<param name="publish_rate" type="double" value="$(arg publish_rate)" />
		<param name="wheel_radius" type="double" value="$(arg wheel_radius)" />
		<param name="front_to_rear_wheel_distance" type="double" value="$(arg front_to_rear_wheel_distance)" />
		<param name="left_to_right_wheel_distance" type="double" value="$(arg left_to_right_wheel_distance)" />
		<param name="min_steering" type="double" value="$(arg min_steering)" />
		<param name="max_steering" type="double" value="$(arg max_steering)" />
	</node>

</launch>
